nntrainer/graph/network_graph.cpp

```diff
@@ -364,7 +364,7 @@ void NetworkGraph::backwarding(
   int iteration,
   std::function<void(std::shared_ptr<LayerNode>, int)> &backwarding_op,
   std::function<void(Weight &, int)> &apply_grad_clip_op,
-  std::function<bool(void *userdata)> stop_cb) const {
+  std::function<bool(void *userdata, void *stat)> stop_cb) const {
```

Comment:
I don't think we need this. User can define it in their userdata. It could make the API more complicate.

Reply:
Remove it